AdminUi/apps/admin_ui/lib/home/identities_overview/identities_data_table_source.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:admin_api_sdk/admin_api_sdk.dart';
+import 'package:data_table_2/data_table_2.dart';
+import 'package:flutter/material.dart';
+import 'package:get_it/get_it.dart';
+import 'package:intl/intl.dart';
+import 'package:logger/logger.dart';
+
+class IdentityDataTableSource extends AsyncDataTableSource {
+  Pagination? _pagination;
+  var _sortingSettings = (sortColumnIndex: 0, sortAscending: true);
+
+  IdentityOverviewFilter? _filter;
+
+  void sort({required int sortColumnIndex, required bool sortColumnAscending}) {
+    _sortingSettings = (sortColumnIndex: sortColumnIndex, sortAscending: sortColumnAscending);
+    notifyListeners();
+  }
+
+  String _getFieldNameByIndex(int index) => switch (index) {
+        0 => 'address',
+        2 => 'createdWithClient',
+        3 => 'numberOfDevices',
+        4 => 'createdAt',
+        5 => 'lastLoginAt',
+        6 => 'datawalletVersion',
+        7 => 'identityVersion',
+        _ => throw Exception('Invalid column index')
+      };
+
+  String _getODataOrderBy() {
+    return '${_getFieldNameByIndex(_sortingSettings.sortColumnIndex)} ${_sortingSettings.sortAscending ? 'asc' : 'desc'}';
+  }
```

Comment:
the method was more readable before, I think you got my last comment wrong, please return to the "old coding" :)